helpers/HTMLView.js

```diff
@@ -91,6 +95,148 @@ export function getCallbackCodeString(jsFunctionName: string, commandName: strin
 `
 }
 
+
+/**
+ * Convert a note's content to HTML and include any images as base64
+ * @author @cwhittl
```

Comment:
Really?

Reply:
You mean because i wrote a few lines it's not my code?  I 100% don't know what I was thinking, removed.